taichi/backends/vulkan/runtime.cpp

```diff
@@ -157,7 +158,7 @@ class HostDeviceContextBlitter {
     }
 
     if (require_sync) {
-      device_->get_compute_stream()->submit_synced(cmdlist);
+      device_->get_compute_stream()->submit_synced(cmdlist, wait_semaphore);
```

Comment:
Do we need to use the sema returned by `submit_synced`?

Reply:
Not in this case. In the case where we are on multiple queues than `submit_synced` only syncs that one queue, and we need the semaphore for cross queue sync